local/jimm/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Installing delve in the image to avoid running go install in running containers
+FROM cosmtrek/air:latest AS dev
+
+RUN go install github.com/go-delve/delve/cmd/dlv@latest
+
+ENTRYPOINT [ "air" ]
+
+# Create a target without air that we use in smoke tests.
+FROM golang:latest as smoke
```

Comment:
should we instead go for a specific version? e.g. golang:1.24.1-alpine, so that we're aligned with out go.mod?